main.css

```diff
@@ -3,3 +3,36 @@
 @tailwind components;
 
 @tailwind utilities;
+
+@responsive {
+  .rotate-0 {
+    transform: rotate(0deg);
+  }
+  .rotate-90 {
+    transform: rotate(90deg);
+  }
+  .rotate-180 {
+    transform: rotate(180deg);
+  }
+  .rotate-270 {
+    transform: rotate(270deg);
+  }
+  .rotate-45 {
+    transform: rotate(45deg);
+  }
+  .-rotate-15 {
+    transform: rotate(-15deg);
+  }
+}
+
+@responsive {
+  .text-shadow {
+    text-shadow: 2px 2px #ff0000;
+  }
+}
+
+@responsive {
+  .linear {
```

Comment:
Irrelevant CSS classname.
Maybe name it like `linear-gradient-primary` as you used the primary colors.

Reply:
done